src/components/cc-tcp-redirection-form/cc-tcp-redirection-form.js

```diff
@@ -1,11 +1,14 @@
 import { css, html, LitElement } from 'lit';
```

Comment:
**nitpick:** within this component, the docs link is still present within the desc. Since there's a link within the footer, I would remove it:
![image](https://github.com/user-attachments/assets/7cddc7e9-f750-43c4-9d23-7ad6f2a78a3e)